packages/wallet-sdk/src/sign/scw/SCWSigner.ts

```diff
@@ -78,30 +84,93 @@ export class SCWSigner implements Signer {
     this.handleResponse(args, decrypted);
   }
 
-  async request(request: RequestArguments) {
+  async initSubAccountConfig() {
+    const config = store.subAccountsConfig.get();
+
+    if (!config?.enableAutoSubAccounts) {
+      throw new Error('called initSubAccountConfig but enableAutoSubAccounts is false');
+    }
+
+    // Get the owner account
+    const { account: owner } = config.toOwnerAccount
+      ? await config.toOwnerAccount()
+      : await getCryptoKeyAccount();
+
+    if (!owner) {
+      throw standardErrors.provider.unauthorized('No owner account found');
+    }
+
+    // Set the capabilities for the sub account
+    const capabilities = {
+      addSubAccount: {
+        account: {
+          type: 'create',
+          keys: [
+            {
+              type: owner.address ? 'address' : 'webauthn-p256',
+              key: owner.address || owner.publicKey,
```

Comment:
Have we tested this when using a local account?

Reply:
i added some options for testing local accounts in the page add-sub-account page in #1554, but will double check